### PR TITLE
Port To TelepathyGLib and Python 3

### PR DESCRIPTION
**Pull request by Creatune**
@quozl @walterbender @pro-panda @tchx84 @chimosky Please let me know if there are any pending changes.

### CODE REVIEW

**Comment by Creatune** [comment]
Should I make seperate PRs for port to TelepathyGLib and python 3 or is this fine?

**Comment by Aniket21mathur** [comment]
@Creatune did you tested these changes? I reviewed, and not able to figure out how `from collabwrapper.collabwrapper import CollabWrapper` will work?

**Comment by Creatune** [comment]
Yes I did test. Currently I claimed the port to TelepathyGLib task, so I thought that I can focus on correcting the port to python 3 later. I got an error regarding the import of collabwrapper that there is no such module named 'collabwrapper'. I will try to fix it :) (I guess I just have to install the module)

**Comment by Creatune** [comment]
Should I create a separate PR for porting to python 3?

File: messenger.py [hunk 1]
```diff
@@ -50,9 +50,9 @@ def serialize(self):
             'oid': self.oid})
 
 
-class Messenger(ExportedGObject):
+class Messenger(CollabWrapper):
```
Comment (Aniket21mathur) [inline messenger.py:53]: @Creatune I don't understand these changes are these changes required? Also, the `telepathy` port seems incomplete. 
The `activity.py` file requires more changes.

**Comment by Aniket21mathur** [comment]
> Should I create a separate PR for porting to python 3?

You can do it in this pr also, it is up to you. But It would be great if we complete the telepathy part before the porting part :-)

File: messenger.py (hunk 1 above)
Comment (Creatune) [reply messenger.py:53]: I will have to learn how collabwrapper works. For now, what I did was just a guess. I will get back to that later :)

**Comment by Creatune** [comment]
I will create seperate PRs for python and telepathy port. This PR is confusing to test.
